packages/pn-pa-webapp/src/pages/PrivacyPolicy.page.tsx

```diff
@@ -33,11 +31,9 @@ const PrivacyPolicyPage = () => {
   }, []);
 
   void waitForElement('.otnotice-content').then(() => {
```

Comment:
we can move this into the `OneTrust.NoticeApi.Initialized.then` callback. Do for all the apps